sros/docker/launch.py

```diff
@@ -827,10 +826,22 @@ def __init__(self, hostname, username, password, mode, variant_name, conn_mode):
             # LC VM Instantiation
             start_eth = 1
             lc_slot_tracker = []
+
             for lc in variant["lcs"]:
-                lc_slot = lc["slot"]
+                lc_slot = lc.get("slot", None)
```

Comment:
maybe lets have a fallback to `1`? 
```suggestion
                lc_slot = lc.get("slot", "1")
```